### DISCUSSION TITLE
Is there an example for function-free deployment template?

### DISCUSSION THREAD

**Discussion by EricPyZhou** [serverless/serverless#12209] [Q&A] [answered]
I am using the template for gcp services. Later I found out that I only wanted to deploy resources rather than functions. I do not find such an example in the repo. Currently, I tried a few ways of modification e.g. leaving functions section empty, and commenting on all package. functions section, but none of them worked.

for now, my workaround is to create a dummy function.

**Comment by EricPyZhou** [comment]
Considering it has been two months, Is there any feedback on the question?

**Comment by edercarloscosta** [comment, marked as answer]
Hi @EricPyZhou ,

Answering by my own experience, seems like the only resource you can update separately are functions i.e `sls deploy -f <function_name>`.

There is a relation with CloudFormation that requires updating the full stack. 

I already spent a lot of time on this topic without success.

**Reply by EricPyZhou** [reply]
@edercarloscosta  Still thanks for helping out!

**Comment by EricPyZhou** [comment]
> Hi @EricPyZhou ,
> 
> Answering by my own experience, seems like the only resource you can update separately are functions i.e `sls deploy -f <function_name>`.
> 
> There is a relation with CloudFormation that requires updating the full stack.
> 
> I already spent a lot of time on this topic without success.
